lib/dev/ui/spinner.rb

```diff
@@ -31,7 +31,7 @@ class Task
           def initialize(title, &block)
             @title = title
             @thread = Thread.new do
-              cap = Dev::UI::StdoutRouter::Capture.new(with_frame_inset: false, &block)
+              cap = Dev::UI::StdoutRouter::Capture.new(self, with_frame_inset: false, &block)
```

Comment:
This is so the block gets the task as a parameter, so you can call `#update_title` on it.